### ISSUE TITLE
ND doesn't contain american/chicago

### ISSUE THREAD

**Issue by paultag** [unitedstates/python-us#15]

**Comment by paultag** [comment]
(thanks @shanear)

**Comment by jcarbaugh** [comment]
ND's current list is: `America/North_Dakota/Center,America/North_Dakota/Beulah,America/North_Dakota/Center,America/North_Dakota/New_Salem`

@shanear I suppose I should add both `America/Chicago,America/Denver` since it includes central and mountain?

**Comment by shanear** [comment]
yeah, not sure what the source of truth on this is, but the IANA has these timezones in ND: 

```
America/Boise
America/Chicago
America/North_Dakota/Beulah
America/North_Dakota/Center
America/North_Dakota/New_Salem
```

see: https://en.wikipedia.org/wiki/Time_in_North_Dakota
